lib/connectors/SentinelConnector/index.ts

```diff
@@ -104,7 +106,12 @@ export default class SentinelConnector extends AbstractConnector {
         }
         if (resolved) {
           debug('resolved: %s:%s', resolved.host, resolved.port)
-          _this.stream = createConnection(resolved)
+          if (_this.options.enableTLSForSentinelMode && _this.options.tls) {
+            Object.assign(resolved, _this.options.tls)
```

Comment:
Shouldn't `Object.assign(resolved, _this.options.tls)` be `Object.assign(resolved, {tls: _this.options.tls})`?

Reply:
Hahaha, that's what I thought too and I spent so much time trying to figure out why it wasn't working!!

https://nodejs.org/api/tls.html#tls_tls_connect_options_callback

> secureContext: TLS context object created with tls.createSecureContext(). If a secureContext is not provided, one will be created by passing the entire options object to tls.createSecureContext().


The entire options object is passed to `createSecureContext()`.

Reply:
The `StandardConnector` works the same: https://github.com/luin/ioredis/blob/master/lib/connectors/StandaloneConnector.ts#L45-L47